gammapy/image/utils.py

```diff
@@ -260,11 +282,13 @@ def exclusion_distance(exclusion):
 
     Parameters
     ----------
-    TODO
+    exclusion : array_like
```

Comment:
If you look at the implementation of the function in scipy that accepts `array_like`, it probably calls `np.asanyarray` at the start?
I don't want to have 1000s of lines of this boilerplate code in Gammapy.

The only place where it makes sense is in functions that end-users use interactively a lot, like
```
>>> from gammapy.stats import significance
>>> significance([2,3], [4,5])
```
When used interactively, users will pass in lists and it should work.

Putting something in single backticks tells Sphinx that it's a link to a function or class, e.g.
```
The `gammapy.stats.significance` function ...
```
will generate a link in the HTML version.

Adding a `~` at the front means that the text displayed will be just "significance" and not "gammapy.stats.significance".

Maybe you could look through the Astropy and Gammapy docs a bit and play with this yourself if you're interested to understand it fully ... but it's also something that you'll pick up over time.